matter_server/server/device_controller.py

```diff
@@ -56,19 +55,18 @@ def __init__(
         self._nodes: dict[int, MatterNode] = {}
         self.wifi_credentials_set: bool = False
         self.thread_credentials_set: bool = False
+        self.compressed_fabric_id: int | None = None
```

Comment:
I think we need to adapt the `get_info` method in the server to check for optional value after this change.

Reply:
Well that is a theoretical discussion because get-info can not be called without the stack being initialized (and thius the compressed_fabric_id known). For now I've just added an assertion there.

Reply:
Yes, that's enough.